ui/raidboss/data/triggers/unending_coil_ultimate.js

```diff
@@ -212,185 +221,241 @@
     { id: 'UCU Nael Quote 1',
       regex: /From on high I descend, in blessed light to bask/,
       regexFr: /Des cieux je vais descendre et révérer la lune/,
+      regexDe: /Seht, ich steige herab, vom rotglühenden Monde/,
+      regexJp: /我、舞い降りて<br>月を仰がん/,
       infoText: {
         en: 'Spread => In',
         fr: 'Se dispercer => Dedans',
+        de: 'Verteilen => Rein',
       }, 
       durationSeconds: 6,
       tts: {
         en: 'spread then in',
         fr: 'Se dispercer, puis dedans',
+        de: 'verteilen, dann rein',
       },
     },
     { id: 'UCU Nael Quote 2',
       regex: /From on high I descend, mine enemies to smite/,
       regexFr: /Du haut des cieux, je vais descendre pour conquérir/,
+      regexDe: /Seht, ich steige herab, um euch zu beherrschen/,
+      regexJp: /我、舞い降りて<br>鉄の覇道を征く/,
       infoText: {
         en: 'Spread => Out',
         fr: 'Se dispercer => Dehors',
+        de: 'Verteilen => Raus',
       },
       durationSeconds: 6,
       tts: {
         en: 'spread then out',
-        fr: "Se dispercer, puis dehors",
+        fr: 'Se dispercer, puis dehors',
+        de: 'verteilen, dann raus',
       },
     },
     { id: 'UCU Nael Quote 3',
       regex: /O refulgent moon, shine down your light/,
       regexFr: /Baignez dans la bénédiction de la lune incandescente/,
+      regexDe: /Flammender Pfad, geschaffen vom roten Mond/,
+      regexJp: /赤熱せし<br>月の祝福を/,
       infoText: {
         en: 'Stack => In',
         fr: 'Se rassembler => Dedans',
+        de: 'Stack => Rein',
       },
       durationSeconds: 6,
       tts: {
         en: 'stack then in',
         fr: 'Se rassembler, puis dedans',
+        de: 'stek dann rein',
       },
     },
     { id: 'UCU Nael Quote 4',
       regex: /Blazing path, lead me to conquest/,
       regexFr: /La voie marquée par l'incandescence mène à la domination/,
+      regexDe: /Umloderter Pfad, führe mich zur Herrschaft/,
+      regexJp: /赤熱し、焼かれし道を<br>鉄の覇道と成す/,
       infoText: {
         en: 'Stack => Out',
         fr: 'Se rassembler => Dehors',
+        de: 'Stack => Raus',
       },
       durationSeconds: 6,
       tts: {
         en: 'stack then out',
-        fr: "Se rassembler, puis dehors",
+        fr: 'Se rassembler, puis dehors',
+        de: 'stek dann raus',
       },
     },
     { id: 'UCU Nael Quote 5',
       regex: /O red moon, scorch mine enemies/,
       regexFr: /Que l'incandescence de la lune brûle mes ennemis/,
+      regexDe: /O roter Mond! Umlodere meinen Pfad/,
+      regexJp: /月よ！<br>赤熱し、神敵を焼け/,
```

Comment:
Hmm, surely it doesn't say <br> in Japanese.  I wonder what that ends up looking like in game.  Maybe just replace those with `.*`?

Reply:
I guess those are in there since as it seems those lines are written in html and they wanted to start a new paragraph for each line in the Japanese version. 

I've tried to look up japanese ACT triggers for UCoB though and as it seems in the JP version they just insert the last paragraph of the text, as it's different for every mechanic and that br seems to cause ACT issues when reading the logs. Gonna correct that :)